django/thunderstore/repository/package_upload.py

```diff
@@ -143,9 +144,13 @@ def save(self, *args, **kwargs):
         self.instance.file_size = self.file_size
         team = self.cleaned_data["team"]
         team.ensure_can_upload_package(self.user)
+        # We just take the namespace with team name for now
+        namespace = Namespace.objects.get_or_create(
```

Comment:
Well now that we have a function for getting the team's namespace, you might as well use it
```python
namespace = team.get_namespace()
```

Reply:
This also means the `get_namespace` implementation in a team should create it if it doesn't exist, or error if it does exist but doesn't belong to the team (contrary to what I mentioned in last review). But it should still only use the name of the team rather than taking a dynamic value for namespace name, if that makes sense.